build-logic-commons/basics/src/main/kotlin/gradlebuild/basics/accessors/accessors.kt

```diff
@@ -19,18 +19,11 @@ package gradlebuild.basics.accessors
 import org.gradle.api.Project
 import org.gradle.api.file.SourceDirectorySet
 
-import org.gradle.api.tasks.GroovySourceSet
-import org.gradle.api.tasks.SourceSet
-
 import org.gradle.kotlin.dsl.*
 import org.jetbrains.kotlin.gradle.dsl.KotlinJvmProjectExtension
 
 
 // TODO these accessors should be generated - https://github.com/gradle/gradle/issues/3191
 
-
-val SourceSet.groovy: SourceDirectorySet
```

Comment:
Idea suggests that this removal will break https://github.com/gradle/gradle/pull/28051/files#diff-8205f80069f1c76e244e78865eddbdddd106f1b319f86bd876b8abfe12efa42aR83

Reply:
Well, it doesn't ;) I need to apply the Groovy plugin, though. Did you re-sync?

Reply:
> Did you re-sync?

Yes I did, although not with the entire changeset.